torch/distributed/algorithms/_checkpoint/checkpoint_wrapper.py

```diff
@@ -219,13 +245,16 @@ def apply_activation_checkpointing(
         their checkpoint-wrapped modules.
     Note::
         This function will not wrap the overall root module. If this is needed, please directly use
-        :class:`CheckpointWrapper`.
+        :func:`checkpoint_wrapper` or :func:`offload_wrapper`.
     Usage::
         model = nn.Sequential(
             nn.Linear(10, 10), nn.Linear(10, 10), nn.Linear(10, 10)
         )
         check_fn = lambda l: isinstance(l, nn.Linear)
+        # Checkpoint activations
         apply_activation_checkpointing(model, checkpoint_wrapper_fn=checkpoint_wrapper, check_fn=check_fn)
+        # Or Offload activations to CPU
+        apply_activation_checkpointing(model, checkpoint_wrapper_fn=offload_wrapper, check_fn=check_fn)
```

Comment:
I think having the name `apply_activation_checkpointing()` as the entry point for both `checkpoint_wrapper` and `offload_wrapper` may be confusing since it suggests only the former.

It may be better to rename it to something like `apply_activation_wrapper()` to make it clear that it can be used for both activation _checkpointing_ and _offloading_ and it is done so via a module wrapper. (We can decide on a different name as well.)